tests/test_models/test_mutables/test_diffchoiceroute.py

```diff
@@ -44,13 +47,24 @@ def test_forward_arch_param(self):
 
         new_diff_choice_route = MODELS.build(new_diff_choice_route_cfg)
 
-        arch_param = new_diff_choice_route.build_arch_param()
+        mutator.prepare_from_supernet(new_diff_choice_route)
```

Comment:
Use user-defined instead  of `mutator`  to construct `arch_param`, as we hope the mutable could work(build and forward) independently in our design.